ollama/_client.py

```diff
@@ -186,7 +188,7 @@ def generate(
     context: Optional[Sequence[int]] = None,
     stream: Literal[False] = False,
     raw: bool = False,
-    format: Optional[Literal['', 'json']] = None,
+    format: Optional[Union[Literal['', 'json'], Type[BaseModel]]] = None,
```

Comment:
This should also accept arbitrary Python types by way of `pydantic.TypeAdapter` as well as the `TypeAdapter` itself

Reply:
Never mind